### PR TITLE
fix: remove the sugar/parse directory

### PR DESCRIPTION
**Pull request by natenichols**
## High Level Overview of Change
Removes old code from the `sugar/parse` directory.

### CODE REVIEW

**Review by mvadari — COMMENTED** [review]
will review when tests are passing

File: src/sugar/orderbook.ts [hunk 1]
```diff
@@ -81,7 +84,7 @@ async function getOrderbook(
   const buy: BookOffer[] = []
   const sell: BookOffer[] = []
   orders.forEach((order) => {
-    if (order.Flags === OfferCreateTransactionFlags.tfSell) {
+    if (order.Flags === SELL_OFFER_FLAGS) {
```
Comment (mvadari) [inline src/sugar/orderbook.ts:87]: This shouldn't be `===` - that doesn't account for the possibility that multiple flags are turned on

**Review by mvadari — APPROVED** [review]

File: src/sugar/orderbook.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ import {
   TakerAmount,
 } from '../models/methods/bookOffers'
 
-import { orderFlags } from './parse/flags'
+const SELL_OFFER_FLAGS = 0x00020000
```
Comment (mvadari) [inline src/sugar/orderbook.ts:12]: Is it not possible to get this number from one of the `Flags` things that we already have? Seems bad to have it as a separate constant.

Comment (natenichols) [reply src/sugar/orderbook.ts:12]: Right, so these flags are `lsf` flags. The flags we have defined are `tf` flags. 

So,  we can define these flags in the `./src/models/ledger` directory with the associated ledger entry.

Comment (mvadari) [reply src/sugar/orderbook.ts:12]: I'd just add a TODO for now then - not worth doing in this PR

File: src/models/ledger/offer.ts [hunk 3]
```diff
@@ -16,3 +16,8 @@ export default interface Offer extends BaseLedgerEntry {
   PreviousTxnLgrSeq: number
   Expiration?: number
 }
+
+export enum OfferLedgerFlags {
+  lsfPassive = 0x00010000,
+  lsfSell = 0x00020000, // True, offer was placed as a sell.
```
Comment (mvadari) [inline src/models/ledger/offer.ts:22]: need the comments above the line for the linter to be happy

**Review by JST5000 — APPROVED** [review]
LGTM!

File: src/sugar/orderbook.ts (hunk 2 above)
Comment (natenichols) [reply src/sugar/orderbook.ts]: Just added it.

**Pull request merged**